modules/installation-mirror-repository.adoc

```diff
@@ -22,38 +22,41 @@ Complete the following steps on the bastion host:
 . Review the
 link:https://access.redhat.com/downloads/content/290/[{product-title} downloads page]
 to determine the version of {product-title} that you want to install.
+The image release advisory includes the pullspec repository and digest, for example link:https://access.redhat.com/errata/product/290/ver=4.3/rhel---7/x86_64/RHBA-2020:0062[`quay.io/openshift-release-dev/ocp-release` and `sha256:3a516480dfd68e0f87f702b4d7bdd6f6a0acfdac5cd2e9767b838ceede34d70d` for the x86-64 4.3.0 release image].
```

Comment:
```suggestion
The image release advisory includes the pullspec repository and digest. For example, the link:https://access.redhat.com/errata/product/290/ver=4.3/rhel---7/x86_64/RHBA-2020:0062[put the title of the page that loads] release uses `quay.io/openshift-release-dev/ocp-release` for the pullspec repository and  `sha256:3a516480dfd68e0f87f702b4d7bdd6f6a0acfdac5cd2e9767b838ceede34d70d` for the x86-64 4.3.0 release image digest.
```

The portal is down, so I can't confirm the link title. It's better to put the title of the page in the link text and explain what you're getting outside the link.